src/Console/Util/PhpProcess.php

```diff
@@ -63,7 +63,23 @@ public function start(callable $callback = null, array $env = null): void
         $phpConfig = new PhpConfig();
 
         $phpConfig->useOriginal();
+
+        // As of 1.3.2 xdebug-handler won't update $_ENV if it is in use.
+        // But Symfony's Process will happily import everything from $_ENV,
+        // hence we need to reset it just as xdebug-handler does
```

Comment:
Seems like xdebug-handler not just unsets, but set the key to some value:

https://github.com/composer/xdebug-handler/blob/0af624bc9c20a3c9252793625bbe976af082d401/src/Process.php#L153-L157

What is the value of `$unset` variable here when the bug is reproduced? (`true` / `false`)?

Reply:
We also set it later to the correct value.